AnnoyancesFilter/sections/popups.txt

```diff
@@ -124,6 +126,7 @@ indianexpress.com###id_newsletter_subscription_popup
 hibboux.com##.fix-wapp
 senpa.io###new-panel
 ||macmost.com/wp-content/themes/*/awpop-subscribe-*.js
+||cyprusjobcentre.com/template/jl/jQuery.NagAds.min.js
```

Comment:
Change to this `||cyprusjobcentre.com/template/*/jQuery.NagAds.min.js`, also remove cosmetic rules.